Makefile

```diff
@@ -162,14 +163,12 @@ build-ia-operator-reference:
 			--env SSH_AUTH_SOCK=/ssh-agent\
 			-w /go/src/$(PACKAGE_NAME) \
 			$(CALICO_BUILD) /bin/bash -c '$(GIT_CONFIG_SSH) rm -rf builder && mkdir builder && cd builder && \
-				git clone --depth=1 -b ia_kb_v2 https://github.com/Brian-McM/gen-crd-api-reference-docs api-gen && cd api-gen && \
-				go mod edit -replace github.com/tigera/image-assurance=github.com/$(IA_OPERATOR_REPO)@$(IA_OPERATOR_VERSION) && \
-				go mod tidy && go mod download all && go build && \
-				./gen-crd-api-reference-docs \
-					-api-dir github.com/tigera/image-assurance/operator/api \
+				git clone --depth=1 -b $(API_GEN_VERSION) git@github.com:ahmetb/gen-crd-api-reference-docs.git && cd gen-crd-api-reference-docs && go build && \
```

Comment:
This comment is not within scope for this PR, but I wonder if we should consider switching off of `gen-crd-api-reference-docs` to one of the alternatives: 
1. Kubernetes reference-docs generator
2. crd-ref-docs by Elastic

It's suggested here on the repo, since the author doesn't have enough time to regularly maintain it: 
- https://github.com/ahmetb/gen-crd-api-reference-docs?tab=readme-ov-file#alternatives

The first alternative is apparently the version of the tool used in the official Kubernetes reference docs.

Reply:
We could consider that, but I'm just trying to fix what we have and use already.

Reply:
> We could consider that, but I'm just trying to fix what we have and use already.

Yep, it's a bigger conversation. Probably only worth it when there is a need to change things (e.g. if `gen-crd-api-reference-docs` gets too out of date and one of the alternatives has a build feature we need).